tools/spectral/ipa/rulesets/IPA-104.yaml

```
@@ -115,3 +115,5 @@ rules:
       function: 'IPA104ValidOperationID'
       functionOptions:
         methodName: 'get'
+        ignoreList:
```

```suggestion
        ignorePluralizationList:
```
[Nit] Name suggestion, I feel like `ignoreList` can be interpreted as operation id ignore list.

---

Will go back and change now